### PR TITLE
Added support for custom hostnames.

### PR DESCRIPTION
**Pull request by Velok**
Had some issues with remote access to the server. The server was listen on localhost. Now it is possible to change the hostname in the brunch-config file with `config.server.hostname`.

### CODE REVIEW

**Comment by goshakkk** [comment]
Probably a CLI param would work better for that, instead of config? The same as `port`

**Comment by Velok** [comment]
With pushserve alone i know its possible to pass a host param. But i want to use `brunch watch` along with the `auto-reload-brunch` plugin.

File: lib/server.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ const startServer = (config, callback) => {
   if (callback == null) callback = Function.prototype;
   const serverOpts = config.server || {};
   const port = parseInt(serverOpts.port, 10);
+  const host = config.server.hostname || 'localhost';
```
Comment (goshakkk) [inline lib/server.js]: What I mean is, can this be getting from `brunch s -h whatever` and not app config? There probably could be a way to access that from auto-reload-brunch... A thing like this, doesn't feel like it belongs into app config

Comment (Velok) [inline lib/server.js]: I tried to find a way to pass a cli arg but i was not successful. Im new in node-coding this project and git and all this exciting stuff anyways. But my first PullRequest at all has passed.. yay! :-) Thanks @paulmillr !

**Comment by paulmillr** [comment]
Thanks Pascal. This is really useful!

**Pull request merged**